ci_framework/roles/operator_build/tasks/clone.yml

```diff
@@ -24,3 +24,15 @@
     repo: "https://github.com/{{ cifmw_operator_build_org }}/{{ operator.name }}.git"
     dest: "{{ operator.src }}"
   when: not stat_op_src.stat.exists
+
+- name: "{{ operator.name }} - Fetch operator src commit"
```

Comment:
IIRC the role allow to pass down a specific commit - we may want to re-use that param if it's set, imho